### ISSUE TITLE
DetectInStream returns "Status=400 Code="BadArgument" Message="JSON parsing error."

### ISSUE THREAD

**Issue by nobiinu-and** [Azure/azure-sdk-for-go#934]
How to reproduce.
run program below.

```
package main

import (
	"context"
	"io"
	"log"
	"net/http"
	"os"

	"github.com/Azure/azure-sdk-for-go/services/cognitiveservices/v1.0/face"
)

func main() {

	client := face.NewClient(face.{Your Region})
	client.Authorizer = autorest.NewCognitiveServicesAuthorizer("{Your Subscription key}")

	image, err := os.Open("{Your ImageFile}")

	if err != nil {
		log.Println("os.Open", err)
		return
	}

	ctx := context.Background()
	returnFaceID := true
	returnFaceLandmark := true
	returnFaceAttributes := []face.AttributeTypes{face.AttributeTypesAge}

	result, err := client.DetectInStream(ctx, image, &returnFaceID, &returnFaceLandmark, returnFaceAttributes)

	if err != nil {
		log.Println("error", err)
		return
	}

	for _, detectedFace := range *result.Value {
		log.Println(*detectedFace.FaceAttributes.Age)
	}
}
```

client should add Content-Type "application/octet-stream" to request, but it seems doesn't add.

I think add `autorest.AsContentType("application/octet-stream")` in [face.go#L172](https://github.com/Azure/azure-sdk-for-go/blob/master/services/cognitiveservices/v1.0/face/face.go#L172)

**Comment by jhendrixMSFT** [comment]
Fixed in v14.0.0.

**Comment by nobiinu-and** [comment]
Thanks a lot!
